detection/face_detector.py

```diff
@@ -0,0 +1,93 @@
+
+import cv2
+import torch
+
+from models.experimental import attempt_load
+from utils.general import check_img_size, non_max_suppression, scale_coords, xyxy2xywh
+from utils.datasets import letterbox
+import copy
+
+from pathlib import Path
+import os
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+IMAGE_SIZE = 640
+CONF_TRESHOLD = 0.3
+IOU_THESHOLD = 0.5
+BASE_PATH = Path(os.path.dirname(os.path.abspath(__file__)))
+
+def get_coors(xywh, w, h):
+    x1 = int(xywh[0] * w - 0.5 * xywh[2] * w)
+    y1 = int(xywh[1] * h - 0.5 * xywh[3] * h)
+    x2 = int(xywh[0] * w + 0.5 * xywh[2] * w)
+    y2 = int(xywh[1] * h + 0.5 * xywh[3] * h)
+    return [x1, y1, x2, y2]
```

Comment:
Why do we need this function? I think the YoloV5 bounding box output format is already x1, y1, x2, and y2 just making an extra effort for nothing? please check accordingly and remove this function.

Reply:
Reviews Done in [b333bd9](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/b333bd972b0250753f031e29491cf0db2a105ce1) and [76b1304](https://github.com/MuhammadAmir5670/intelligent-covid-SOP-compliance/pull/1/commits/76b13040a4b36a906e298232437019c8a653987c)